api/src/main/resources/fireCloud.yaml

```diff
@@ -17,7 +17,7 @@ info:
     url: http://opensource.org/licenses/BSD-3-Clause
   termsOfService: https://github.com/broadinstitute/firecloud-orchestration
 
-host: "firecloud-orchestration.dsde-dev.broadinstitute.org"
+host: "api.firecloud.org"
```

Comment:
What about `notebooks.yaml`? Will it work if it's not in sync with FC?

Reply:
I believe we also do need to swap `notebooks.yaml`.

Reply:
The path is `notebooks.firecloud.org`

Reply:
Also there are two hardcoded references here: ui/src/app/views/workspace/component.ts

Reply:
Updated.